service/build.gradle

```diff
@@ -40,22 +43,25 @@ dependencies {
 	implementation 'org.apache.commons:commons-lang3:3.12.0'
 
 	// Test
-	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	testImplementation("org.springframework.boot:spring-boot-starter-test") {
+		exclude group: "com.jayway.jsonpath", module: "json-path"
+	}
+	testImplementation("com.jayway.jsonpath:json-path:2.5.0") {
+		exclude group: "net.minidev", module: "json-smart"
+	}
+	testImplementation 'net.minidev:json-smart:2.4.1'
 	testImplementation 'org.junit.jupiter:junit-jupiter-params:5.7.0'
-	testImplementation 'com.github.tomakehurst:wiremock:2.27.2'
+	testImplementation "com.github.tomakehurst:wiremock:2.27.2"
```

Comment:
Minor: could we be consistent with the quotes? all `'` or all `"`

Reply:
refactor build gradle to use '